<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SystemServer</class>
 <widget class="QMainWindow" name="SystemServer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SystemServer</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#SystemServer{background-image: url(:/new/prefix1/ui_image/menu.jpg);}

QPushButton {
	background-color: #68ACF9;
    color: white;
    padding: 5px;
    border-radius: 15px;
	color: rgba(255, 255,255, 250);background-color: rgba(0, 33, 68,50);
}

QPushButton:hover {
    background-color: #2F42B5;
}

QPushButton:pressed, QPushButton:checked {
    background-color: #2F42B5;
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="title_lbl">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>安防管理系统</string>
    </property>
   </widget>
   <widget class="click_label" name="monitor_lbl">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>20</y>
      <width>84</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>实时监控</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
   </widget>
   <widget class="click_label" name="photo_lbl">
    <property name="geometry">
     <rect>
      <x>347</x>
      <y>20</y>
      <width>84</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>录像照片</string>
    </property>
   </widget>
   <widget class="click_label" name="log_lbl">
    <property name="geometry">
     <rect>
      <x>484</x>
      <y>20</y>
      <width>84</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>报警信息</string>
    </property>
   </widget>
   <widget class="click_label" name="manager_lbl">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>20</y>
      <width>84</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>系统管理</string>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>70</y>
      <width>1020</width>
      <height>530</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0, 33, 68,10);
</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="monitorAll_widget">
     <widget class="click_label" name="monitor_1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>500</width>
        <height>250</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/new/prefix1/ui_image/menu2.jpg);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="click_label" name="monitor_2">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>20</y>
        <width>500</width>
        <height>250</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/new/prefix1/ui_image/menu1.jpg);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="click_label" name="monitor_4">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>270</y>
        <width>500</width>
        <height>250</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/new/prefix1/ui_image/menu2.jpg);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="click_label" name="monitor_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>270</y>
        <width>500</width>
        <height>250</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/new/prefix1/ui_image/menu1.jpg);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="monitorOne_widget">
     <widget class="QPushButton" name="get_photo_bt">
      <property name="geometry">
       <rect>
        <x>710</x>
        <y>310</y>
        <width>101</width>
        <height>61</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">
QPushButton {
	background-color: #68ACF9;
    color: white;
    padding: 5px;
    border-radius: 15px;
	color: rgba(255, 255,255, 250);background-color: rgba(0, 33, 68,50);
}

QPushButton:hover {
    background-color: #2F42B5;
}

QPushButton:pressed, QPushButton:checked {
    background-color: #2F42B5;
}</string>
      </property>
      <property name="text">
       <string>拍照</string>
      </property>
     </widget>
     <widget class="QPushButton" name="auto_model_bt">
      <property name="geometry">
       <rect>
        <x>710</x>
        <y>420</y>
        <width>101</width>
        <height>61</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">
QPushButton {
	background-color: #68ACF9;
    color: white;
    padding: 5px;
    border-radius: 15px;
	color: rgba(255, 255,255, 250);background-color: rgba(0, 33, 68,50);
}

QPushButton:hover {
    background-color: #2F42B5;
}

QPushButton:pressed, QPushButton:checked {
    background-color: #2F42B5;
}</string>
      </property>
      <property name="text">
       <string>自动模式</string>
      </property>
     </widget>
     <widget class="QPushButton" name="backToMonitorAll_btn">
      <property name="geometry">
       <rect>
        <x>890</x>
        <y>420</y>
        <width>101</width>
        <height>61</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">
QPushButton {
	background-color: #68ACF9;
    color: white;
    padding: 5px;
    border-radius: 15px;
	color: rgba(255, 255,255, 250);background-color: rgba(0, 33, 68,50);
}

QPushButton:hover {
    background-color: #2F42B5;
}

QPushButton:pressed, QPushButton:checked {
    background-color: #2F42B5;
}</string>
      </property>
      <property name="text">
       <string>返回全监</string>
      </property>
     </widget>
     <widget class="QPushButton" name="get_video_bt">
      <property name="geometry">
       <rect>
        <x>890</x>
        <y>310</y>
        <width>101</width>
        <height>61</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">
QPushButton {
	background-color: #68ACF9;
    color: white;
    padding: 5px;
    border-radius: 15px;
	color: rgba(255, 255,255, 250);background-color: rgba(0, 33, 68,50);
}

QPushButton:hover {
    background-color: #2F42B5;
}

QPushButton:pressed, QPushButton:checked {
    background-color: #2F42B5;
}</string>
      </property>
      <property name="text">
       <string>录像</string>
      </property>
     </widget>
     <widget class="QPushButton" name="turn_left_bt">
      <property name="geometry">
       <rect>
        <x>710</x>
        <y>90</y>
        <width>81</width>
        <height>81</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/new/prefix1/ui_image/left.png);
</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="autoRepeat">
       <bool>true</bool>
      </property>
      <property name="autoRepeatDelay">
       <number>30</number>
      </property>
      <property name="autoRepeatInterval">
       <number>30</number>
      </property>
     </widget>
     <widget class="QPushButton" name="turn_down_bt">
      <property name="geometry">
       <rect>
        <x>810</x>
        <y>180</y>
        <width>81</width>
        <height>81</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/new/prefix1/ui_image/down.png);
</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="autoRepeat">
       <bool>true</bool>
      </property>
      <property name="autoRepeatDelay">
       <number>30</number>
      </property>
      <property name="autoRepeatInterval">
       <number>30</number>
      </property>
     </widget>
     <widget class="QPushButton" name="turn_right_bt">
      <property name="geometry">
       <rect>
        <x>910</x>
        <y>90</y>
        <width>81</width>
        <height>81</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/new/prefix1/ui_image/right.png);

</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="autoRepeat">
       <bool>true</bool>
      </property>
      <property name="autoRepeatDelay">
       <number>30</number>
      </property>
      <property name="autoRepeatInterval">
       <number>30</number>
      </property>
     </widget>
     <widget class="QPushButton" name="turn_up_bt">
      <property name="geometry">
       <rect>
        <x>810</x>
        <y>0</y>
        <width>81</width>
        <height>81</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/new/prefix1/ui_image/up.png);
</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="autoRepeat">
       <bool>true</bool>
      </property>
      <property name="autoRepeatDelay">
       <number>30</number>
      </property>
      <property name="autoRepeatInterval">
       <number>30</number>
      </property>
     </widget>
     <widget class="QPushButton" name="turn_reset_bt">
      <property name="geometry">
       <rect>
        <x>820</x>
        <y>100</y>
        <width>61</width>
        <height>61</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/new/prefix1/ui_image/reset.png);
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="video_lbl">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>640</width>
        <height>480</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(0, 33, 68,150);
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="photo_widget">
     <property name="styleSheet">
      <string notr="true">background-color: rgba(0, 33, 68,200);</string>
     </property>
     <widget class="QPushButton" name="previous_btn">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>490</y>
        <width>99</width>
        <height>33</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #68ACF9;
    color: white;
    padding: 5px;
    border-radius: 15px;
	color: rgba(255, 255,255, 250);background-color: rgba(0, 33, 68,50);
}

QPushButton:hover {
    background-color: #2F42B5;
}

QPushButton:pressed, QPushButton:checked {
    background-color: #2F42B5;
}</string>
      </property>
      <property name="text">
       <string>上一页</string>
      </property>
     </widget>
     <widget class="QPushButton" name="next_btn">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>490</y>
        <width>99</width>
        <height>33</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #68ACF9;
    color: white;
    padding: 5px;
    border-radius: 15px;
	color: rgba(255, 255,255, 250);background-color: rgba(0, 33, 68,50);
}

QPushButton:hover {
    background-color: #2F42B5;
}

QPushButton:pressed, QPushButton:checked {
    background-color: #2F42B5;
}</string>
      </property>
      <property name="text">
       <string>下一页</string>
      </property>
     </widget>
     <widget class="QPushButton" name="search_btn">
      <property name="geometry">
       <rect>
        <x>910</x>
        <y>490</y>
        <width>99</width>
        <height>33</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #68ACF9;
    color: white;
    padding: 5px;
    border-radius: 15px;
	color: rgba(255, 255,255, 250);background-color: rgba(0, 33, 68,50);
}

QPushButton:hover {
    background-color: #2F42B5;
}

QPushButton:pressed, QPushButton:checked {
    background-color: #2F42B5;
}</string>
      </property>
      <property name="text">
       <string> 搜索</string>
      </property>
     </widget>
     <widget class="QWidget" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>190</width>
        <height>531</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(0, 33, 68,10);</string>
      </property>
      <widget class="QPushButton" name="video_btn">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>171</width>
         <height>41</height>
        </rect>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	background-color: #68ACF9;
    color: white;
    padding: 5px;
    border-radius: 15px;
	color: rgba(255, 255,255, 250);background-color: rgba(0, 33, 68,50);
}

QPushButton:hover {
    background-color: #2F42B5;
}

QPushButton:pressed, QPushButton:checked {
    background-color: #2F42B5;
}</string>
       </property>
       <property name="text">
        <string>录像</string>
       </property>
      </widget>
      <widget class="QPushButton" name="photo_btn">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>171</width>
         <height>41</height>
        </rect>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	background-color: #68ACF9;
    color: white;
    padding: 5px;
    border-radius: 15px;
	color: rgba(255, 255,255, 250);background-color: rgba(0, 33, 68,50);
}

QPushButton:hover {
    background-color: #2F42B5;
}

QPushButton:pressed, QPushButton:checked {
    background-color: #2F42B5;
}</string>
       </property>
       <property name="text">
        <string>图片</string>
       </property>
      </widget>
      <widget class="QPushButton" name="search_btn_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>171</width>
         <height>41</height>
        </rect>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	background-color: #68ACF9;
    color: white;
    padding: 5px;
    border-radius: 15px;
	color: rgba(255, 255,255, 250);background-color: rgba(0, 33, 68,50);
}

QPushButton:hover {
    background-color: #2F42B5;
}

QPushButton:pressed, QPushButton:checked {
    background-color: #2F42B5;
}</string>
       </property>
       <property name="text">
        <string>搜索</string>
       </property>
      </widget>
     </widget>
     <widget class="QListWidget" name="vidWidget">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>0</y>
        <width>820</width>
        <height>480</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(0, 33, 68,10);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
     </widget>
     <widget class="QListWidget" name="picWidget">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>0</y>
        <width>820</width>
        <height>480</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(0, 33, 68,10);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="del_btn">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>490</y>
        <width>99</width>
        <height>33</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #68ACF9;
    color: white;
    padding: 5px;
    border-radius: 15px;
	color: rgba(255, 255,255, 250);background-color: rgba(0, 33, 68,50);
}

QPushButton:hover {
    background-color: #2F42B5;
}

QPushButton:pressed, QPushButton:checked {
    background-color: #2F42B5;
}</string>
      </property>
      <property name="text">
       <string>删除</string>
      </property>
     </widget>
     <widget class="QPushButton" name="rename_btn">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>490</y>
        <width>99</width>
        <height>33</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #68ACF9;
    color: white;
    padding: 5px;
    border-radius: 15px;
	color: rgba(255, 255,255, 250);background-color: rgba(0, 33, 68,50);
}

QPushButton:hover {
    background-color: #2F42B5;
}

QPushButton:pressed, QPushButton:checked {
    background-color: #2F42B5;
}</string>
      </property>
      <property name="text">
       <string>重命名</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="rename_edt">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>490</y>
        <width>113</width>
        <height>33</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
border:0px;</string>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="search_edt">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>490</y>
        <width>113</width>
        <height>33</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
border:0px;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="log_widget"/>
    <widget class="QWidget" name="manager_widget"/>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>click_label</class>
   <extends>QLabel</extends>
   <header location="global">click_label.h</header>
   <slots>
    <signal>hover()</signal>
    <signal>clicked()</signal>
    <signal>leave()</signal>
    <signal>doubleclicked()</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
